modules/RTC/TraceablePeerConnection.js

```diff
@@ -1899,7 +1899,10 @@ TraceablePeerConnection.prototype.removeTrackFromPc = function(localTrack) {
  * @returns {boolean}
  */
 TraceablePeerConnection.prototype.usesCodecSelectionAPI = function() {
-    return this._usesCodecSelectionAPI;
+    // Browser throws an error when H.264 is set on the encodings. Therefore, munge the SDP when H.264 needs to be
```

Comment:
Is this reported somewhere? A link to a bug we can follow would be nice.

Reply:
I haven't reported it yet, I am not able to reproduce this using a fiddle. I will open a bug report and update it here then.